Packs/SafeNet_Trusted_Access/Integrations/SafeNetTrustedAccess/SafeNetTrustedAccess.py

```diff
@@ -559,6 +560,168 @@ def validate_tenant_sta(self):
             url_suffix='/authorized'
         )
 
+    # Get list of all the applications in the tenant.
+    def get_application_list_sta(self, limit=None):
+
+        uri = '/applications'
+        if limit:
+            limit = self.validate_limit_sta(limit)
+            if limit <= 100:
+                query_params = (
+                    ('pageIndex', 0),
```

Comment:
Is there an option to get to page number 3 using xsoar command?

Reply:
No such parameter/argument for the commands to change the page no.